conanfile.txt

```diff
@@ -5,6 +5,7 @@ cmake/3.29.7
 glm/1.0.1
 stduuid/1.2.3
 gsl/2.7.1
+sdl/2.0.20
```

Comment:
I believe this is up to #15, so even if this pr is ready I think we shall block it until #15 passes.